src/Plugin/Condition/DataComparison.php

```diff
@@ -24,7 +24,8 @@
  *     "operator" = @ContextDefinition("string",
  *       label = @Translation("Operator"),
  *       description = @Translation("The comparison operator."),
- *       required = FALSE
+ *       default_value = "==",
+ *       required = TRUE
```

Comment:
why is it now required? since we have a default value now it should not be required?